spec/intergration/post_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Page', type: :view do
+  describe 'posts#index page' do
+    before(:each) do
+      @user1 = User.create(name: 'Sharon',
+                           photo: "https://images.unsplash.com/photo-1667255949472-2f8bba82deb1?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80\n  ",
+                           bio: 'oooooooo',
+                           postscounter: 0)
```

Comment:
**In your post index page add**

- [ ] Tests that asserts you can see the user's profile picture.
- [ ] Tests that asserts you can see the first comments on a post.
- [ ] Tests that asserts you can see a section for pagination if there are more posts than fit on the vie